examples/06-verif-manual/vm-002-beam_stresses_and_deflections.py

```diff
@@ -0,0 +1,188 @@
+   :math:`M = 10000 \cdot 10 \cdot 60 = 6 \cdot 10^6 lb \cdot in`
+ - Determination of the maximum stress in the middle portion of the beam is:
+   :math:`\sigma_max = \frac{M h}{2 I_z}`
+ - The deflection, :math:`\delta`, at the middle of the beam could be defined
+   by the formulas of the Transversally Loaded Beams:
+   :math:`\delta = 0.182 in`
+"""
+
+# sphinx_gallery_thumbnail_path = '_static/vm2_setup.png'
+
+###############################################################################
+# Start MAPDL
+# ~~~~~~~~~~~
+
+from ansys.mapdl.core import launch_mapdl
+
+# start mapdl and clear it
+mapdl = launch_mapdl()
+mapdl.clear()  # optional as MAPDL just started
+
+# enter verification example mode and the pre-processing routine
+mapdl.verify()
+mapdl.prep7()
+
+###############################################################################
+# Define Element Type
+# ~~~~~~~~~~~~~~~~~~~
+# Set up the element type (a beam-type).
+
+mapdl.antype("STATIC")
+mapdl.et(1, "BEAM188")
+mapdl.keyopt(1, 9, 3)  # Output at 9 intermediate locations
+mapdl.keyopt(1, 3, 3)  # Cubic shape function
+
+###############################################################################
+# Define Material
+# ~~~~~~~~~~~~~~~
+# Set up the material.
+
+mapdl.mp("EX", 1, 30E6)
+mapdl.mp("PRXY", 1, 0.3)
+
+###############################################################################
+# Define Section
+# ~~~~~~~~~~~~~~
+# Set up the cross-section properties for a beam element.
+
+w_f = 1.048394965
+w_w = 0.6856481
+sec_num = 1
+mapdl.sectype(sec_num, "BEAM", "I", "ISection")
+mapdl.secdata(15, 15, 28 + (2 * w_f), w_f, w_f, w_w)
+
+###############################################################################
+# Define Geometry:
+# ~~~~~~~~~~~~~~~~~~~~~~
+# Set up the nodes and elements. Create nodes then creating elements
+# bewtween nodes.
+
+# Define nodes
+mapdl.n(1)
+mapdl.n(5, 480)
+mapdl.n(6, 60, "1 $ N", 10, 420, 1)
+mapdl.fill(1, 5)
+mapdl.fill(6, 10)
+print(mapdl.nlist())  # List of the created nodes.
+
+# Define elements
+
+mapdl.e(1, 2, 6)
+mapdl.egen(4, 1, 1)
+print(mapdl.elist())  # List of the created elements.
+
+# Display elements with their nodes numbers.
+
+mapdl.eplot(show_node_numbering=True, line_width=5, cpos="xy") 
+
+###############################################################################
+# Define Boundary Conditions
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Application of boundary conditions.
+
+mapdl.d(2, "UX", lab2="UY") 
+mapdl.d(4, "UY")
+mapdl.nsel("S", "LOC", "Y", 0)
+mapdl.d("ALL", "UZ")
+mapdl.d("ALL", "ROTX")
+mapdl.d("ALL", "ROTY")
+mapdl.nsel("ALL")
+
+###############################################################################
+# Define Distributed Loads
+# ~~~~~~~~~~~~~~~~~~~~~~~~~
+# Application a distributed force of :math:`w = (10000/12) lb/in` 
+# in the y-direction.
+
+w = 10000/12  # (lb/in) - distributed load.
+mapdl.sfbeam(1, 1, "PRES", w)
+mapdl.sfbeam(4, 1, "PRES", w)
+mapdl.finish()
```

Comment:
Is ``finish`` necessary here?

Reply:
Hi Alex, 
not really. 
At **APDL** when you are writing **"finish"** that means that you really want to be sure that you have stopped all previous procedures. 
But in fact, when you have done all work in **"PREP7"** (Pre-Processing Module) and you're activating **"/SOLU"** (Solution Module) normally all functions from PREP7 can't be activated.
I will remove it.
Thank you.